zktest/src/archive/test2/F86-ZK-3961.zul

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+F86-ZK-3961.zul
+
+        Purpose:
+                
+        Description:
+                
+        History:
+                Thu Jul 12 15:51:26 CST 2018, Created by klyve
+
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+
+-->
+
+<zk>
+	<zscript>
+		public void setHeight300() {
+			knobs.setHeight("300px");
+		}
+		public void setHeight100() {
+			knobs.setHeight("100px");
+		}
+		public void setHeightNull() {
+			knobs.setHeight(null);
+		}
+
+		public void setWidth300() {
+			knobs.setWidth("300px");
+		}
+		public void setWidth100() {
+			knobs.setWidth("100px");
+		}
+
+		public void setAngleArc360() {
+			knobs.setAngleArc(360.0);
+		}
+		public void setAngleArc270() {
+			knobs.setAngleArc(270.0);
+		}
+		public void setAngleArc130() {
+			knobs.setAngleArc(130.0);
+		}
+
+		public void setStrokeWidth20() {
+			knobs.setStrokeWidth(20.0);
+		}
+		public void setStrokeWidth10() {
+			knobs.setStrokeWidth(10.0);
+		}
+	</zscript>
+	<div>
+		<label multiline="true">
+			1. click the knob, it will change the current position.
+			2. do mousedown and move the mouse, it will change the current position, and the input value should be changed.
+			3. wheel the mouse, it will change the current position, and the input value should be changed.
+			4. click button label as "set Angle 270", the angle curve should be as 270 degree, and input should inside the arc curve.
+			5. click button label as "set Angle 140", the angle curve should be as 140 degree, and input should inside the arc curve. 
+			6. click button label as "set SW 20", the stroke width of the curve should be as 20.
+			7. click button label as "set H 300", the height of the widget should be 300px, and the input position should not be changed.
+		</label>
+		<slider id="knobs" mode="knob" minpos="0.0" maxpos="100.0" curpos="40.0" step="0.5" width="300px"/>
```

Comment:
It's **mold** not mode. And `mode="decimal"` or `mode="integer"` can be used with knob mold.

Reply:
ok